script.js

```diff
@@ -0,0 +1,200 @@
+                  EnvironmentScoret.innerText=" Environment Rating " ;
+
+                  //OverallScore
+                  var OS=document.createElement('div');
+                  OS.classList.add('os');
+
+                  var OverallScore=document.createElement('span');
+                  OverallScore.classList.add('scoreo');
+
+                if(resultbrands.result[index].OverallScore){
+                  OverallScore.innerText=resultbrands.result[index].OverallScore;
+                  OverallScore.style["background-color"] =resultbrands.colourMap[resultbrands.result[index].OverallScore];
+                }
+                else {
+                  OverallScore.innerText="TBD";
+                  OverallScore.style["background-color"] =resultbrands.colourMap[resultbrands.result[index].OverallScore];
+                }
+
+                  var OverallScoret=document.createElement('span');
+                  OverallScoret.classList.add('scoret');
+                  OverallScoret.innerText= " Overall Rating ";
+
+                  //Link Page
+                  var brandpagelink ="https://wellspent-ethical.herokuapp.com/#/Brand/"+brand[0];
+                  var brandpage=document.createElement('a');
+                  brandpage.setAttribute("href",brandpagelink);
+                  brandpage.setAttribute("target","_blank")
+                  brandpage.innerText="Click here for more detail";
+                  brandpage.classList.add('brandpage');
+
+
+                  myModal.appendChild(content);
+                  content.appendChild(span);
+                  content.appendChild(score);
+                  content.appendChild(header);
+
+                  OS.appendChild(OverallScore);
+                  OS.appendChild(OverallScoret);
+                  ES.appendChild(EnvironmentScore);
+                  ES.appendChild(EnvironmentScoret);
+                  LS.appendChild(LaborScore);
+                  LS.appendChild(LaborScoret);
+                  TS.appendChild(TransparencyScore);
+                  TS.appendChild(TransparencyScoret);
+
+
+                  content.appendChild(OS);
+                  content.appendChild(ES);
+                  content.appendChild(LS);
+                  content.appendChild(TS);
+
+                  content.appendChild(brandpage);
+
+                  document.body.appendChild(myModal);
+                  myModal.style.display = "block";
+                    }}
+                span.onclick = function() {
+                      myModal.style.display = "none";
+                    }
+                window.onclick = function(event) {
+                      if (event.target == myModal) {
+                      myModal.style.display = "none";
+                      }
+                    }
+            }
+          });
+      }})
+    });
+  })
+  .catch(() =>
+        console.log("Can’t access " + url + " resultbrandsonse. Blocked by browser?")
```

Comment:
"Can’t access " + url + " resultbrandsonse. Blocked by browser?"
At first, I did not understand what is the meaning of this word "resultbrandsonse" in the comment and if the app fails where I should be looking exactly.
Now I can see it's only the name of the variable that contains the response :p